lib/types.js

```diff
@@ -1254,30 +1252,28 @@ module.exports = function(expect) {
     ['Int', 'Uint'].forEach(intOrUint => {
       const constructorName = `${intOrUint + numBits}Array`;
       const Constructor = global[constructorName];
-      if (typeof Constructor !== 'undefined') {
-        expect.addType({
-          name: constructorName,
-          base: 'binaryArray',
-          hexDumpWidth: 128 / numBits,
-          digitWidth: numBits / 4,
-          identify(obj) {
+      expect.addType({
+        name: constructorName,
+        base: 'binaryArray',
+        hexDumpWidth: 128 / numBits,
+        digitWidth: numBits / 4,
+        identify:
+          Constructor !== 'undefined' &&
+          function(obj) {
```

Comment:
More curiosity than anything, is the check outside to avoid being captured by the closure?

Reply:
No, it's just a small perf optimization so we end up with `identify: false` ("abstract" type) instead of having the hot `identify` function check each time.